FightPandemics/Features/Feed/CreatePost/CreatePostFooter.swift

```diff
@@ -0,0 +1,172 @@
+//
+//  CreatePostFooter.swift
+//  FightPandemics
+//
+//  Created by ehsan sat on 5/13/20.
+//
+//  Copyright (c) 2020 FightPandemics
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+
+import UIKit
+
+class CreatePostFooter: UIView {
+    private var sepLine = UIView()
+    private var questionLbl = UILabel()
+    private var shareBtn = UIButton()
+    private var durationBtn = UIButton()
+    private var tagsLbl = UILabel()
+    private var tagsBtn = UIButton()
+    private var postBtn = UIButton()
+    func setUp () {
+        setUpUpper()
+        setUpLower()
+    }
+    func setUpUpper () {
+        // seperator line
+        sepLine.backgroundColor = UIColor(hexString: "#D7D7D7")
+        sepLine.frame.size.width = UIScreen.main.bounds.width
+        sepLine.frame.size.height = 2
+        self.addSubview(sepLine)
+        // first question label
+        questionLbl.text = "Who can see this post and for how long?"
+        questionLbl.textAlignment = .left
+        questionLbl.frame.size.width = UIScreen.main.bounds.width - 47
+        questionLbl.frame.size.height = 22
+        questionLbl.font = Fonts.poppinsBold.customFont(size: 16)
+        questionLbl.numberOfLines = 0
+        self.addSubview(questionLbl)
+        // share button
+        shareBtn.setTitle("Share with anyone", for: .normal)
+        shareBtn.setImage(#imageLiteral(resourceName: "chevron-down"), for: .normal)
+        shareBtn.titleLabel?.font = Fonts.dmSansRegular.customFont(size: 12)
+        shareBtn.clipsToBounds = true
+        shareBtn.backgroundColor = UIColor(hexString: "#F3F4FE")
```

Comment:
maybe we should add the palette to the project to avoid using the hexString

Reply:
long term thats a good abstraction to introduce for sure - doesn't need to be within this PR